Source/utils/pcx.cpp

```diff
@@ -1,5 +1,6 @@
 #include "utils/pcx.hpp"
 
+#include <SDL_endian.h>

```

Comment:
Looks like you cherry-picked, or rebased master on top of your branch, rather then the other way around?

Reply:
I may have. I'll ping you on Discord.